easybuild/tools/options.py

```diff
@@ -474,6 +474,8 @@ def config_options(self):
                        None, 'store', None),
             'recursive-module-unload': ("Enable generating of modules that unload recursively.",
                                         None, 'store_true', False),
+            'recursive-module-unload-depends-on': ("Use depends_on (Lmod 7.6.1+) for dependencies in all generated modules.",
```

Comment:
Should this just be `use-depends-on`, and then have the help message clarify that this implies that it makes unloading of modules recursive?